octavia_proxy/tests/functional/api/v2/base.py

```diff
@@ -103,6 +109,89 @@ def tearDown(self):
             self._sdk_connection.close()
         super().tearDown()
 
+    def _get_sdk_connection(self):
+        global _sdk
+        if not _sdk:
+            _sdk = openstack.connect()
+        return _sdk
+
+    def _create_network(self):
+        global _network
+        cidr = '192.168.0.0/16'
+        ipv4 = 4
+        uuid_v4 = uuidutils.generate_uuid()
+        router_name = 'octavia-proxy-test-router-' + uuid_v4
+        net_name = 'octavia-proxy-test-net-' + uuid_v4
+        subnet_name = 'octavia-proxy-test-subnet-' + uuid_v4
+
+        if not _network:
+            if not self._sdk_connection:
+                self._sdk_connection = self._get_sdk_connection()
+            network = self._sdk_connection.network.create_network(name=net_name)
+            net_id = network.id
+            subnet = self._sdk_connection.network.create_subnet(
+                name=subnet_name,
+                ip_version=ipv4,
+                network_id=net_id,
+                cidr=cidr
+            )
+            subnet_id = subnet.id
+
+            router = self._sdk_connection.network.create_router(name=router_name)
+            router_id = router.id
+            interface = router.add_interface(
+                self._sdk_connection.network,
+                subnet_id=subnet_id
+            )
+            _network = {
+                'router_id': router_id,
+                'subnet_id': subnet_id,
+                'network_id': net_id
+            }
+        return _network
+
+    def _destroy_network(self, params: dict):
```

Comment:
I actually start thinking now - creating net on first need is not a problem. But how are we going to figure out when it is not required anymore (with multiple test modules) and clean it up?